.github/workflows/tests.yml

```
@@ -11,21 +11,21 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        python-version: ["3.8", "3.10", "3.11"]
+        python-version: ["3.8"]
         django: ["32"]
```

According to [this](https://github.com/openedx/edx-platform/blob/76330b36b004f1636908ada962bbc75f4857eeb3/.github/workflows/unit-tests.yml#L20) and taking into account that Quince starts using Django 4.2, I suggest...

```suggestion
        django: ["32","42"]
```

---

Thank you very much!! I've already made the corrections.